EIPS/eip-1485.md

```diff
@@ -1,7 +1,7 @@
 ---
 eip: 1485
 title: TEthashV1
-author: trustfarm (KT Ahn - 안씨아저씨) <trustfarm.info@gmail.com>, trustfarm <cpplover@trustfarm.net>
+author: trustfarm KT Ahn - 안씨아저씨 <trustfarm.info@gmail.com>, trustfarm <cpplover@trustfarm.net>

```

Comment:
I'm not sure if this is the best way to handle this. I'm not sure what the author was hoping to convey using the parentheses.

Reply:
I would probably talk to the author but my gut says to just remove the stuff in parenthesis.  It seems like that email address is a company owned address, and the person on the other end of that address is subject to change.  Authors should not be subject to change, so I think just `trustfarm` is appropriate.

Reply:
Updated.